Missing return statement in function returning non-void

in `ext/libev/ev_iouring.c`, `iouring_sqe_submit` and `iouring_internal_destroy` function returning non-void but no return statement, cause build failed with


```
../libev/ev_iouring.c:300:1: error: no return statement in function returning non-void [-Werror=return-type]
  300 | }
      | ^
../libev/ev_iouring.c: In function ‘iouring_internal_destroy’:
../libev/ev_iouring.c:331:1: error: no return statement in function returning non-void [-Werror=return-type]
  331 | }
```

OS: openSUSE tumbleweed

--- a/ext/libev/ev_iouring.c
+++ b/ext/libev/ev_iouring.c
@@ -287,7 +287,7 @@ iouring_sqe_get (EV_P)
 }
 
 inline_size
-struct io_uring_sqe *
+void
 iouring_sqe_submit (EV_P_ struct io_uring_sqe *sqe)
 {
   unsigned idx = sqe - EV_SQES;
@@ -313,7 +313,7 @@ iouring_tfd_cb (EV_P_ struct ev_io *w, int revents)
 
 /* called for full and partial cleanup */
 ecb_cold
-static int
+static void
 iouring_internal_destroy (EV_P)
 {
   close (iouring_tfd);

